packages/@sanity/base/src/change-indicators/helpers/getOffsetsTo.ts

```diff
@@ -25,33 +33,15 @@ const getOffsetsTo = (source: HTMLElement, target: HTMLElement) => {
       bounds.height = el.offsetHeight
       bounds.left = el.offsetLeft
       bounds.width = el.offsetWidth
+
       foundScrollContainer = true
     }
-    top += el.offsetTop - el.scrollTop
-    left += el.offsetLeft - el.scrollLeft
-    el = el.offsetParent as HTMLElement
-  }
 
-  return {
-    rect: {
-      top,
-      left,
-      height: source.offsetHeight,
-      width: source.offsetWidth,
-    },
-    bounds: bounds,
-  }
-}
+    rect.top += el.offsetTop - el.scrollTop
+    rect.left += el.offsetLeft - el.scrollLeft
 
-const time = <Args, Ret>(label, fn: (...args: Args[]) => Ret): ((...args: Args[]) => Ret) => {
-  return (...args: Args[]) => {
-    // eslint-disable-next-line no-console
-    console.time(label)
-    const ret = fn(...args)
-    // eslint-disable-next-line no-console
-    console.timeEnd(label)
-    return ret
+    el = el.offsetParent as HTMLElement
   }
-}
 
-export default getOffsetsTo
```

Comment:
we gotta add [this one](https://github.com/import-js/eslint-plugin-import/blob/7c382f02a6cafe676ee754a896092040dad8daf6/docs/rules/no-default-export.md) eventually